CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Updated Open Enclave to [0.19.2](https://github.com/openenclave/openenclave/releases/tag/v0.19.2).
 - Updated Open Enclave to [0.19.3](https://github.com/openenclave/openenclave/releases/tag/v0.19.3).
 - Expose COSESign1 `content` for `user_cose_sign1` authenticated endpoints in JavaScript/TypeScript apps (#5465).
+- Add HMAC support to JS API. Call with `ccf.crypto.sign({"name": "HMAC", "hash": "SHA-256"}, key, data)`.
```

Comment:
We ought to update https://github.com/microsoft/CCF/blob/b44e90aa38c2d50b621effe617103e8996aa9cd0/js/ccf-app/src/global.ts#L217, or possible split the type (but probably just update, since this is using sign() too).

Reply:
Also good to check the polyfill and update the corresponding test if possible.